### PR TITLE
initial Shelly2 support

### PR DESCRIPTION
**Pull request by eurich**
Shelly 2 in relay mode added.

Tested: 
-  manual switches both work as expected
- relays both work as expected
- Reverting back to stock firmware works as expeted..

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -22,6 +22,9 @@ Shelly1PM: build-Shelly1PM
 ShellyPlugS: build-ShellyPlugS
 	@true
 
+Shelly25: build-Shelly2
```
Comment (kzyapkov) [inline Makefile]: duplicate target name here, you probably meant `Shelly2`

File: mos.yml [hunk 2]
```diff
@@ -1,6 +1,6 @@
 author: 'Deomid "rojer" Ryabkov'
 description: A HomeKit firmware for Shelly switches
-version: 1.3.3
+version: 1.3.4
```
Comment (rojer) [inline mos.yml]: this is new functionality, so make it 1.4.0 (https://semver.org/)

**Comment by rojer** [comment]
thank you for contibuting! please address Kiril's comment and updater version and we're good to go.

**Comment by eurich** [comment]
should be fixed now :)

**Comment by rojer** [comment]
thank you!

**Comment by rojer** [comment]
pushed firmware zip file

**Pull request merged**